Update support_04_store_protect.md (#76)
DHP-stottepakke/pages: pages/dmp_supporting_info/support_04_store_protect.md

If specific solutions are chosen during data processing and analysis or preservation, this may be specified in the respective chapters.

## Question-specific guidance

> ### Storage guidelines

Reviewer: Should we indicate if a question is e.g. marked with HPC?

Author: Don't think so, we might not show the tags in the end(?)

Reviewer: Might be confusing to users to see guidance on questions they cannot see?

Author: Are the colors for the tags fixed?
We could use the following:
![Static Badge](https://img.shields.io/badge/DSW-ELSI-%23D35400)
![Static Badge](https://img.shields.io/badge/DSW-HPC-%23CED4DA)
![Static Badge](https://img.shields.io/badge/DSW-Horizon%20Europe-%232ECC71)
![Static Badge](https://img.shields.io/badge/DSW-IT%20security%20Europe-%23BDC3C7)
![Static Badge](https://img.shields.io/badge/DSW-RCN%2FScience%20Europe%20security%20Europe-%232980B9)
![Static Badge](https://img.shields.io/badge/DSW-cross%20institutional%20project-%23C0392B)
![Static Badge](https://img.shields.io/badge/DSW-machine%20actionable-%239B59B6)
![Static Badge](https://img.shields.io/badge/DSW-minimum-%23F39C12)
![Static Badge](https://img.shields.io/badge/DSW-umbrella%20DMP-%233498DB)

Reviewer: Only four tags with user relevance: SE, Minimum, ELSI, HPC? Colors are not fixed, yet.

For some domains there might be existing guides to estimate the data volume generated by instruments. (e.g. for [Life Science data: Illumina  Approximate sizes of sequencing run output folders](https://support.illumina.com/bulletins/2018/01/approximate-sizes-of-sequencing-run-output-folders.html), [Publication: Numerical Compression Schemes for Proteomics Mass Spectrometry Data](https://doi.org/10.1074/mcp.O114.037879))

### If applicable: Will storage need change over time?

> In many cases the storage needs of a project will increase with project length. Taking this into consideration will help with budgeting. Also in some cases there might be a need for temporary storage of intermediate results, which do not have to be stored for longer time-frames.

Reviewer: ```suggestion
In many cases the storage needs of a project will increase with project length. Taking this into consideration will help with budgeting and reducing expenses for storage space. Also in some cases there might be a need for temporary storage of intermediate results, which do not have to be stored for longer time-frames.
```

How the members of your project access the data and how access is controlled can be crucial. The overall capacity and transfer speed of a storage solution can be critical parameters in case you have larger amounts of data.
You should also consider if a more advanced way to organise your data, such as object storage or databases might be of help for your project.

### Information security level(s) this storage solution will be used for

> While the general information security levels used at Norwegian higher educational institutions are similar (green, yellow, red, black), the classification between which data falls into which category and which solution is allowed for which security level might vary. Please consult the [institutional storage guides](support_04_store_protect#storage-guidelines) for details.

Reviewer: Suggest to repeat links for user-friendlyness.

Author: ```suggestion
While the general information security levels used at Norwegian higher educational institutions are similar (green, yellow, red, black), the classification between which data falls into which category and which solution is allowed for which security level might vary. Please consult the institutional storage guides for details:

* [NTNU Data storage guide](https://i.ntnu.no/wiki/-/wiki/English/Data+storage+guide)
* [UiB Storage guide](https://www.uib.no/en/foremployees/153608/storage-guide)
* [UiO Data storage guide](https://www.uio.no/english/services/it/security/lsis/storage-guide.html)
* [UiT Research data portal - Processing and storage](https://en.uit.no/research/research-dataportal/art?p_document_id=729174)
```
I think repeating the block might break the reading flow, but would leave it to you @jennyostrop

### Is the storage solution associated with costs?

In case your project requires storage beyond the basic supplied level by your institution, there might be associated costs. Often it is good practise to familiarize your self with storage costs upfront, so that they can be factored into your budget. In many cases the costs will vary depending on the needs and nature of your project and potentially by funding source. If you are using commercial cloud storage also consider that traffic in and out of the system might be charged.

### Backups

If you are using institutional solutions, the institution often provides backup and snapshotting functionality. It is good to know about the frequency and limitations of these before you need to recover data in your project. If you are using other solutions you might have to check the backup routines of theses solutions or have to implement you own routines.

> An often suggested backup rule is 3-2-1.

Reviewer: ```suggestion
If not relying on managed storage, an often suggested backup rule is 3-2-1.
```

> * External Copy: Store a third copy in a cloud solution or a central storage.
>
> Advantages of the 3-2-1 Backup Rule
> * Security: Data is protected from loss through physical separation and redundancy.
> * Access: Researchers can access their data even if one backup medium fails.
> * Traceability: Regular backups and the ability to access older versions of the data enable quick detection and correction of errors and data corruption.
>
> What the 3-2-1 Rule Does Not Include
>
> The 3-2-1 rule does not include a backup routine. However, this is just as important because the best backup is useless if it is out of date. Automations (e.g., a cron job) on your servers are best suited for this, copying your data to the secondary storage every night. Or use central services, they already have backup plans.

### File naming and organisation
Consistent file naming and folder structures are an important part of organising data in your research project. It is critical to be consistent and to make all of your project partners following the same convention.

> There are different strategies for file naming and folder structure. Please consult the chapters in e.g. the [RDMkit for life sciences or the CESSDA Data Management Guide](support_04_store_protect#further-resources)  for inspiration.

Reviewer: ```suggestion
There are different strategies for file naming and folder structure. Please consult the chapters in e.g. the [RDMkit for life sciences or the CESSDA Data Management Guide](support_04_store_protect#further-resources) for inspiration.
```
